node/pkg/db/governor.go

```diff
@@ -138,6 +142,16 @@ func UnmarshalTransfer(data []byte) (*Transfer, error) {
 		t.Hash = string(hash[:n])
 	}
 
+	if err := binary.Read(reader, binary.BigEndian, &t.TargetChain); err != nil {
+		return nil, fmt.Errorf("failed to read target chain id: %w", err)
+	}
+
+	targetAddress := vaa.Address{}
```

Comment:
Is vaa.Address() the emitter address?

Reply:
`vaa.Address` is the type used to hold the various addresses (emitter, origin and token).